### PR TITLE
Snhut 1992 flagged assessment state

### PR DESCRIPTION
**Pull request by shafi-sadaindia**
https://collectiveshift.atlassian.net/browse/SNHUT-1994
- As an assessor,I need the ability to remove a flag from an assessment, So I can continue grading the assessment. 
https://collectiveshift.atlassian.net/browse/SNHUT-1992
- As an assessor,I need the ability to identify which assessments are flagged, So that I can focus on non-flagged assessments

- [JIRA Ticket](https://collectiveshift.atlassian.net/browse/SNHUT-1994)
- [JIRA Ticket](https://collectiveshift.atlassian.net/browse/SNHUT-1992)

Screenshot 2022-11-22 at 9 30 42 PM
Screenshot 2022-11-22 at 9 30 55 PM
Screenshot 2022-11-22 at 9 31 10 PM

#### Checklist

- [x] **Assumptions of User Story met (including acceptance criteria)**
- [ ] **All new necessary Unit tests were CREATED** to promote code coverage
- [ ] **All required unit tests are PASSING**
- [x] The proposed functionality was tested manually locally, and it's working fine.
- [ ] PR was properly reviewed by a team member (at least 1 approval required to merge!)
- [x] Unnecessary console.logs AND/OR comments were removed
- [x] Feature ok-ed by UX designer (If needed - not for back end components) and Product Owner
- [x] Ensure configuration and/or build changes documented

### CODE REVIEW

**Review by joao-sd — APPROVED** [review]
LGTM

File: apps/toboggan-app/src/app/assessment/components/assessment-list/assessment-list.component.scss [hunk 1]
```diff
@@ -0,0 +1,7 @@
+::ng-deep {
```
Comment (artem-sada) [inline apps/toboggan-app/src/app/assessment/components/assessment-list/assessment-list.component.scss:1]: @shafi-sadaindia Could you please check mine implementation of flagged row styling?
Perhaps we could synchronize our implementations

https://github.com/snhuproduct/toboggan-ws/pull/110/files#diff-f7b891787dfdbc4ebc79e6e21dfb0cdec0b8aff29dc4c349fa685ba1d39de485R101-R108 
https://github.com/snhuproduct/toboggan-ws/pull/110/files#diff-f7b891787dfdbc4ebc79e6e21dfb0cdec0b8aff29dc4c349fa685ba1d39de485R80-R81
https://github.com/snhuproduct/toboggan-ui-components/pull/264/files#diff-513b0e144e3fb5f26327b8cc1e61108773007e30cf09fbe4d3eeb6e2a83ea810R204-R224

Comment (shafi-sadaindia) [reply apps/toboggan-app/src/app/assessment/components/assessment-list/assessment-list.component.scss:1]: Sure @artem-sada

Comment (shafi-sadaindia) [reply apps/toboggan-app/src/app/assessment/components/assessment-list/assessment-list.component.scss:1]: @artem-sada I will take your code and work once you complete PR.

**Pull request closed without merging**